docker-images/oncoanalyser/assets/nextflow_aws_template.config

```diff
@@ -12,21 +12,21 @@ wave {
 
 aws {
   batch {
-    jobRole = 'arn:aws:iam::843407916570:role/OncoanalyserStack-OncoanalyserTaskBatchInstanceRol-DXWG0PGDIBDY'
+    jobRole = '__BATCH_INSTANCE_ROLE__'
     volumes = '/mnt/local_ephemeral/:/tmp/'
   }
 }
 
 params {
   genomes {
     GRCh38_umccr {
-      fasta           = "s3://umccr-research-dev/stephen/oncoanalyser_data/genomes/GRCh38_umccr/GRCh38_full_analysis_set_plus_decoy_hla.fa"
-      fai             = "s3://umccr-research-dev/stephen/oncoanalyser_data/genomes/GRCh38_umccr/samtools_index/1.16/GRCh38_full_analysis_set_plus_decoy_hla.fa.fai"
-      dict            = "s3://umccr-research-dev/stephen/oncoanalyser_data/genomes/GRCh38_umccr/samtools_index/1.16/GRCh38_full_analysis_set_plus_decoy_hla.fa.dict"
-      bwa_index       = "s3://umccr-research-dev/stephen/oncoanalyser_data/genomes/GRCh38_umccr/bwa_index/0.7.17-r1188/"
-      bwa_index_image = "s3://umccr-research-dev/stephen/oncoanalyser_data/genomes/GRCh38_umccr/bwa_index_image/0.7.17-r1188/GRCh38_full_analysis_set_plus_decoy_hla.fa.img"
-      gridss_index    = "s3://umccr-research-dev/stephen/oncoanalyser_data/genomes/GRCh38_umccr/gridss_index/2.13.2/GRCh38_full_analysis_set_plus_decoy_hla.fa.gridsscache"
-      star_index      = "s3://umccr-research-dev/stephen/oncoanalyser_data/genomes/GRCh38_umccr/star_index/gencode_38/2.7.3a/"
+      fasta           = "__S3_HMF_GENOMES_DATA_PATH__/GRCh38_umccr/GRCh38_full_analysis_set_plus_decoy_hla.fa"
+      fai             = "__S3_HMF_GENOMES_DATA_PATH__/GRCh38_umccr/samtools_index/1.16/GRCh38_full_analysis_set_plus_decoy_hla.fa.fai"
+      dict            = "__S3_HMF_GENOMES_DATA_PATH__/GRCh38_umccr/samtools_index/1.16/GRCh38_full_analysis_set_plus_decoy_hla.fa.dict"
+      bwa_index       = "__S3_HMF_GENOMES_DATA_PATH__/GRCh38_umccr/bwa_index/0.7.17-r1188/"
+      bwa_index_image = "__S3_HMF_GENOMES_DATA_PATH__/GRCh38_umccr/bwa_index_image/0.7.17-r1188/GRCh38_full_analysis_set_plus_decoy_hla.fa.img"
+      gridss_index    = "__S3_HMF_GENOMES_DATA_PATH__/GRCh38_umccr/gridss_index/2.13.2/GRCh38_full_analysis_set_plus_decoy_hla.fa.gridsscache"
+      star_index      = "__S3_HMF_GENOMES_DATA_PATH__/GRCh38_umccr/star_index/gencode_38/2.7.3a/"
```

Comment:
Since these reference files aren't Hartwig specific can `__S3_HMF_GENOMES_DATA_PATH__` be changed to `__S3_GENOMES_DATA_PATH__`?

Reply:
Resolved in 7acc9c13da2d7887fd0065cbb5f7971d1f22d4dc